text/src/autogluon/text/automm/data/process_text.py

```diff
@@ -260,13 +331,21 @@ def build_one_token_sequence_from_text(
             "Token indices sequence length is longer than.*result in indexing errors"
         )
         for col_name, col_text in text.items():
+            #if training
```

Comment:
May I ask why we have parentheses () for the if statements? I didn't see such usage in other places at AG. Besides, sometimes there are extra spaces (like `if( self.train_augmenter is not None  )`), sometimes there are missing spaces (like ` if(is_training):` and ` #if augment`). Consistency would make the code nicer to read, thanks.

Reply:
@lzcemma let’s remove that. Also, remove the comments